SRC/FileOperation.cpp

```diff
@@ -121,58 +123,69 @@ int FileOperation::getLinesCount()
 
 void FileOperation::CreateDB_Folder()
 {
-	string cmd = "mkdir " + database_folder;
-	system(cmd.c_str());
-	cmd = "mkdir " + database_folder + "/" + file_name + "_db";
-	system(cmd.c_str());
+	string cmd = CMDMKDIR + string(SPACE) + database_folder + SEMICOLON + CMDMKDIR + string(SPACE) + database_folder + file_name + DB_EXT;

```

Comment:
mkdir fails , abort the mutation with valid info.

Reply:
Used boost lib for the same and changes are raised in new PR